loader/reset.go

```diff
@@ -40,6 +41,21 @@ func (p *ResetProcessor) UnmarshalYAML(value *yaml.Node) error {
 
 // resolveReset detects `!reset` tag being set on yaml nodes and record position in the yaml tree
 func (p *ResetProcessor) resolveReset(node *yaml.Node, path tree.Path) (*yaml.Node, error) {
+	// If the path contains "<<", removing the "<<" element and merging the path
+	if strings.Contains(path.String(), "<<") {
```

Comment:
isn't this just equivalent to `path = strings.Replace(path, ".<<", "")` ?

Reply:
Yes, I have made this change.